devops/scripts/benchmarks/benches/gromacs.py

```diff
@@ -0,0 +1,344 @@
+# Copyright (C) 2025 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+import subprocess
+import tarfile
+import urllib.request
+from pathlib import Path
+from .base import Suite, Benchmark
+from options import options
+from utils.utils import git_clone
+from utils.result import Result
+
+
+class GromacsBench(Suite):
+    GROMACS_REPO = "https://gitlab.com/gromacs/gromacs.git"
+    GROMACS_TAG = "v2025.1"
+    GRAPPA_BENCHMARKS_URL = (
+        "https://zenodo.org/record/11234002/files/grappa-1.5k-6.1M_rc0.9.tar.gz"
+    )
+
+    def __init__(self, directory):
+        # Initialize GromacsBench-specific attributes
+        self.directory = Path(directory).resolve()
+        self.gromacs_dir = self.directory / "gromacs"
+        self.grappa_dir = self.directory / "grappa-1.5k-6.1M_rc0.9"
+        self.build_dir = self.gromacs_dir / "build"
+
+    def name(self):
+        return "Gromacs Bench"
+
+    def benchmarks(self) -> list[Benchmark]:
+        systems = [
+            "0001.5",
+            "0003",
+            "0006",
+            "0012",
+            "0024",
+            "0048",
+            "0096",
+            "0192",
+            "0384",
+        ]
+        return [
+            GromacsSystemBenchmark(self, system, self.gromacs_dir, self.grappa_dir)
+            for system in systems
+        ]
+
+    def setup(self):
+        print(f"Working directory: {self.directory}")
+        self.directory.mkdir(parents=True, exist_ok=True)
+
+        if not self.gromacs_dir.exists():
+            print(
+                f"Cloning GROMACS repository (tag: {self.GROMACS_TAG}) into {self.gromacs_dir}..."
+            )
+            repo_path = git_clone(
+                self.directory,
+                "gromacs-repo",
+                self.GROMACS_REPO,
+                self.GROMACS_TAG,
+            )
+            print(f"GROMACS repository cloned to {repo_path}")
+        else:
+            print(f"GROMACS repository already exists at {self.gromacs_dir}")
+
+        # Build GROMACS
+        self.build_dir.mkdir(parents=True, exist_ok=True)
+        print(f"Building GROMACS in {self.build_dir}...")
+        subprocess.run(
```

Comment:
use utils.run method. It adds sycl.
same everywhere. don't use subprocess.run directly.